### ISSUE TITLE
2.10.10 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3783]
- #3782
- #3779 
- #3778 
- #3719

## Pick list
```
git cherry-pick ba082a03de9e3fa910e47e1c1c7b7be3d23aa380
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.10.9..HEAD --pretty='format:- %s'`
```
_Note: Updates Drupal core to 10.2.10. Includes updates that address [SA-CORE-2024-002](https://www.drupal.org/sa-core-2024-002) and [CVE-2024-45440](https://github.com/advisories/GHSA-mg8j-w93w-xjgc)._

- Closes #3718 Improve styling of AZ Finder exposed form, especially for multiple vocabulary filters (#3719)
- Closes #3777: Update drupal/core-recommended to 10.2.10 for SA-CORE-2024-002 and CVE-2024-45440 (#3778)
- #3737: Backport coding standards fixes from #3738 to 2.10.x. (#3779)
- Closes #3780: Add missing SMTP module config to address test failures in 2.10.x (#3782)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.10.10
